test/math.jl

```diff
@@ -1168,6 +1168,12 @@ end
             @test hypot(x, x*f, x*f) ≈ x * hypot(one(f), f, f) rtol=eps(T)
         end
     end
+    @testset "Real and Complex $T" for T in (Float16, Float32, Float64, BigFloat)
```

Comment:
I would be okay with excluding `Float16` here and then having an explicit `@test_broken` case